app/services/curation_concerns/admin_set_service.rb

```diff
@@ -1,7 +1,11 @@
 module CurationConcerns
   class AdminSetService
-    def initialize(user)
+    attr_reader :user, :context
+    delegate :blacklight_config, :repository, to: :context
+
+    def initialize(user, context)
```

Comment:
@cbeer I'm proceeding w/ `CollectionService` based on this branch and the proposed switched param order.